apollo-router/src/configuration/tests.rs

```diff
@@ -170,10 +179,10 @@ subgraphs:
     assert_eq!(
         error.to_string(),
         String::from(
-            r#"configuration had errors: 
+            r#"configuration had errors:
 1. at line 4
 
-  
```

Comment:
Ah, this is the source of the failing tests. Lines are being stripped of trailing whitespace upon saving, which is causing the test's exact string match to fail.